File: js/UberBlack.js
import Car from './Car.js'
class UberBlack extends Car {
    constructor(license, driver, typoCardAcepted, material){
        super(license, driver);
        this.typoCardAcepted = typoCardAcepted;
        this.material = material;
    }
}

export default UberBlack;

File: Python/uberBlack.py
from car import Car
class UberBlack(Car):
    typeCardAcepted = []
    material = []
    def __init__(self,license, driver, typeCardAcepted, material):
        super(UberBlack, self).__init__(license, driver)
        self.typeCardAcepted = typeCardAcepted
        self.material = material